CMakeLists.txt

```diff
@@ -95,9 +101,10 @@ set(CHEMFILES_OBJECTS
     $<TARGET_OBJECTS:molfiles>
     $<TARGET_OBJECTS:fmt::object>
     $<TARGET_OBJECTS:tng_io>
-    $<TARGET_OBJECTS:tng_io_zlib>
     $<TARGET_OBJECTS:msgpackc-object>
     $<TARGET_OBJECTS:mmtf-c>
+    ${ZLIB_OBJECTS}
```

Comment:
This is slightly different than the other `$<TARGET_OBJECTS>`, is this intentional as `${ZLIB_OBJECTS}` can be empty?

Reply:
Yes, this is the same strategy we already use for netcdf

Reply:
Fair enough.